example.rules

```diff
@@ -0,0 +1,9 @@
+# The count of CPUs per node, useful for getting CPU time as a percent of total.
+instance:node_cpu:count = count(node_cpu{mode="idle"}) without (cpu,mode)
+
+# CPU in use by CPU.
+instance_cpu:node_cpu_not_idle:rate2m = sum rate(node_cpu{mode!="idle"}[2m])) without (mode)
```

Comment:
No real reason, it's short enough to catch some CPU use spikes, but long enough for systems configured with a 30s scrape interval.